docs/changehistory/Roadmap.md

```diff
@@ -21,123 +21,129 @@ Volunteers for help on any or all of these projects are welcome. In particular,
-- Auto-updating data in all presentation rules -driven components when:
-  - data in source iModel changes
-  - presentation rules change
-  - presentation ruleset variables change
+- [ ] Filtering presentation rules and rulesets by supported ECSchema version.
+- [ ] Auto-updating data in all presentation rules -driven components when:
+  - [ ] data in source iModel changes
+  - [ ] presentation rules change
+  - [ ] presentation ruleset variables change
 
 #### Tree-related Improvements
 
-- Making tree nodes "favorite" to make them appear at the top of the hierarchy.
-- Improve filtering performance.
-- Excluding or exclusively including some nodes from the hierarchy when filtering.
+- [ ] Making tree nodes "favorite" to make them appear at the top of the hierarchy.
+- [ ] Improve filtering performance.
+- [ ] Excluding or exclusively including some nodes from the hierarchy when filtering.
 
 #### Properties-related Improvements
 
-- Nested property grouping.
-- Clickable navigation properties that select the target element.
-- Better support for composite properties (arrays, structs, points).
-- Support relationship properties.
-- Support ad-hoc properties.
+- [ ] Nested property grouping.
+- [ ] Clickable navigation properties that select the target element.
+- [ ] Better support for composite properties (arrays, structs, points).
+- [ ] Support relationship properties.
+- [ ] Support ad-hoc properties.
 
 ### User Interface
 
-- Extend functionality of panel-based AppUi.
-- Improve touch support and user experience on mobile devices.
-- Multi-window support for desktop Electron applications.
-- Enhance accessibility and keyboard navigation.
-- Enhance UiItemProvider to support:
-  - More type editors, especially those necessary to read/write workflows
-  - Async list population
-- Provide an API to allow apps, extensions, and packages to supply settings to the Settings Stage.
+- [ ] Extend functionality of panel-based AppUi.
+- [ ] Improve touch support and user experience on mobile devices.
+- [ ] Multi-window support for desktop Electron applications.
+- [ ] Enhance accessibility and keyboard navigation.
+- [ ] Enhance UiItemProvider to support:
+  - [ ] More type editors, especially those necessary to read/write workflows
+  - [ ] Async list population
+- [ ] Provide an API to allow apps, extensions, and packages to supply settings to the Settings Stage.
 
 ### Interfaces to external systems
 
-- Support for exporting to common file formats (e.g. gltf, obj, dxf, pdf, etc.)
-- Provide reference implementation of reality data in Unity.
-- Provide more open source examples of integration with third-party visualization systems (Unity, Blender, etc.)
-- Expand filtering and quality control options for graphics extraction.
+- [ ] Support for exporting to common file formats (e.g. glTF, obj, dxf, pdf, etc.)
+  - [X] glTF
+  - [ ] obj
+  - [ ] dxf
+  - [ ] pdf
+- [ ] Provide reference implementation of reality data in Unity.
```

Comment:
@MichaelBelousov @mgooding

Reply:
I'll remove this - outside of iTwin.js scope.

We will have a "reality model to single GLTF" API on developer.bentley.com - that's the simple answer for viewing in Unity. Otherwise, developers should use a third party 3D tiles implementation or write their own.